packages/experiments-realm/app-task-card.gts

```diff
@@ -185,13 +185,9 @@ class AppTaskCardIsolated extends Component<typeof AppTaskCard> {
     }
   });
 
-  @action createNewTask(statusLabel: string) {
-    this.loadingColumnKey = statusLabel;
-    this._createNewTask.perform(statusLabel);
-  }
-
-  @action isCreateNewTaskLoading(statusLabel: string) {
-    return this.loadingColumnKey === statusLabel;
+  @action async onMoveCardMutation(draggedCard: DndItem, newColumn: DndColumn) {
+    // This only updates the value of the card but doesn't commit to fileystem nor server
+    draggedCard.status.label = newColumn.title;
   }
```

Comment:
Intentionally did not do a server-side mutation here. Just updated the field of the card in the frontend. But, it doesn't save to file system or roundtrip to server

Reply:
Actually, this.args.model setting shud roundtrip but it looks like here I can't seem to reach in to this.args.model of cards. So let's just handle that in another PR or discuss first